integration/configs/browserCompatibilityTest/video_test_processor.config.json

```diff
@@ -1,20 +1,22 @@
 {
   "`": 1,
   "name": "video_test_processor.config.json %ts",
-  "grids": [
-  ],
+  "grids": [],
```

Comment:
We should remove this test. It kinda duplicate of the background blur/replacement test and it does not actually test video filter. FYI @nainkunal933

Reply:
Should I just directly delete this file and `video_test_simulcast.config.json`?

Reply:
Should I just remove these tests in `/browserCompatibilityTest/` or also the one in `/integrationTest`?